src/runtime/qurt_hvx.cpp

```diff
@@ -2,6 +2,7 @@
 #include "HalideRuntimeQurt.h"
 #include "printer.h"
 #include "mini_qurt.h"
+#include "hexagon_remote/log.h"
```

Comment:
Does this even work? I really don't think it works, I don't think you can use log_printf from outside the hexagon remote runtime.

debug(user_context) will eventually end up at log_printf. However, I think most of the debugging code here should be removed, especially because debug(...) is not a no-op outside of debug modes.

Finally, preprocessor directives are confusing here. I don't think turning on the debug target flag should lead to a change in the debug behavior of this code.